WFInfo/Main.cs

```diff
@@ -166,7 +166,7 @@ public void ThreadedDataLoad()
         }
         private async void TimeoutCheck()
         {
-            if (!await dataBase.IsJWTvalid().ConfigureAwait(true) || _process.GameIsStreamed)
+            if (!await dataBase.IsJWTvalid().ConfigureAwait(true))
```

Comment:
```suggestion
            if (!await dataBase.IsJWTvalid().ConfigureAwait(true))
```
This is target to change, i'm not yet certain why its excluding GameIsStreamed. Maybe because LogCapture shouldn't be invoked for it.